yt_dlp/downloader/fragment.py

```diff
@@ -21,6 +23,77 @@
 )
 
 
+class Timer:
+    TIMER_FUNC = time.monotonic
```

Comment:
Is there a reason to extracto out `time.monotonic`? We have to use monotonic time anyways since we rely on ordering of timestamps, so what other value could it have? I would inline it instead.